src/NuGet.Core/NuGet.ProjectModel/TargetFrameworkInformation.cs

```diff
@@ -73,7 +73,7 @@ internal TargetFrameworkInformation(TargetFrameworkInformation cloneFrom)
             Imports = new List<NuGetFramework>(cloneFrom.Imports);
             AssetTargetFallback = cloneFrom.AssetTargetFallback;
             Warn = cloneFrom.Warn;
-            DownloadDependencies = CloneList(cloneFrom.DownloadDependencies, static dep => dep.Clone());
+            DownloadDependencies = cloneFrom.DownloadDependencies.ToList();
```

Comment:
Here we still need to clone the mutable collection, but no longer need to clone the elements within it.